docs/UserGuide.md

```diff
@@ -35,9 +39,11 @@ Command Line Interface (CLI)** while still having the benefits of a Graphical Us
 
     * **`delete 1`** : Deletes the first candidate in the system.
 
-    * **`find k/Jane f/name`** : Searches for all candidate with name containing “Jane/jane”.
+    * **`find k/Alex f/name`** : Searches for all candidate with name containing “jane" (case-insensitive).
```

Comment:
Perhaps the `"jane"` should be `alex` instead? 🤔

Reply:
oh yah cos loading from the sample data right! alright lemme make that change!! :-) thank you!!!